packages/flutter_tools/test/general.shard/project_test.dart

```diff
@@ -117,10 +117,6 @@ void main() {
           FlutterProject.fromDirectory(directory).directory.absolute.path,
           directory.absolute.path,
         );
-        expect(
-          FlutterProject.fromPath(directory.path).directory.absolute.path,
```

Comment:
What sort of unit tests for the new API would make sense?

Reply:
Added a test that matched the others - which isn't great coverage. I'll need to take a pass on this file later, at any rate